projects/mycelium/index.js

```diff
@@ -0,0 +1,47 @@
+const sdk = require("@defillama/sdk");
+const { transformArbitrumAddress } = require("../helper/portedTokens");
+const Vault = require("./Vault.json");
+
+const VAULT_ADDRESS = "0xDfbA8AD57d2c62F61F0a60B2C508bCdeb182f855";
+
+async function arbitrumTVL(timestamp, block, chainBlocks) {
+  const balances = {};
+
+  const { output: numTokens } = await sdk.api.abi.call({
+    target: VAULT_ADDRESS,
+    abi: Vault.allWhitelistedTokensLength,
+    chain: "arbitrum",
+    block: chainBlocks["arbitrum"],
+  });
+
+  const { output: tokenAddresses } = await sdk.api.abi.multiCall({
+    abi: Vault.allWhitelistedTokens,
+    calls: Array.apply(null, { length: numTokens }).map((_, i) => ({
+      target: VAULT_ADDRESS,
+      params: [i],
+    })),
+    chain: "arbitrum",
+    block: chainBlocks["arbitrum"],
+  });
+
+  const tokenBalances = await sdk.api.abi.multiCall({
```

Comment:
you can refactor it a bit:

    const Vault = require("./Vault.json");
    const { sumTokens2 } = require('../helper/unwrapLPs')

    const VAULT_ADDRESS = "0xDfbA8AD57d2c62F61F0a60B2C508bCdeb182f855";

    async function arbitrumTVL(timestamp, block, chainBlocks) {
      const chain = 'arbitrum'
      const block = chainBlocks[chain]

      const { output: numTokens } = await sdk.api.abi.call({
        target: VAULT_ADDRESS,
        abi: Vault.allWhitelistedTokensLength,
        chain, block,
      });

      const { output: tokenAddresses } = await sdk.api.abi.multiCall({
        abi: Vault.allWhitelistedTokens,
        calls: Array.apply(null, { length: numTokens }).map((_, i) => ({
          target: VAULT_ADDRESS,
          params: [i],
        })),
        chain, block,
      });

      const tokens = tokenAddresses.map(i => i.output)
      return sumTokens2({ owner: VAULT_ADDRESS, tokens, chain, block,})
    }

    module.exports = {
      arbitrum: {
        tvl: arbitrumTVL,
      }
    }